src/index.js

```diff
@@ -0,0 +1,177 @@
+"use strict";
+
+// const { default: axios } = require("axios");
+
+// const axios = require('axios');
+
+const state = {
+    temp: 23,
+    city: 'Seattle',
+    lat: 47.6038321,
+    lon: -122.3300624,
+    weather: 'rainy',
+};
+
+const addTemp = () => {
+    state.temp += 1;
+    const tempCount = document.querySelector("#temp");
+    tempCount.textContent = `${state.temp}`;
+};
+
+const lowerTemp = () => {
+    state.temp -= 1;
+    const tempCount = document.querySelector("#temp");
+    tempCount.textContent = `${state.temp}`;
+};
+
+const tempThemeToggle = () => {
+    console.log('tempThemeToggle called');
+
+    addTemp();
+    lowerTemp();
+
+    const tempAdjustElement = document.getElementById("tempAdjust");
+    const tempAdjustLandscape = document.getElementById("landScape");
+
+    if (state.temp >= 32){
+        tempAdjustElement.className = "tooHot";
+        tempAdjustLandscape.className = "landHot";
+    } else if (state.temp >= 25 && state.temp < 32){
+        tempAdjustElement.className = "hot";
+        tempAdjustLandscape.className = "landTemperate";
+    } else if (state.temp > 0 && state.temp <= 11) {
+        tempAdjustElement.className = "cold";
+        tempAdjustLandscape.className = "landCold";
+    } else if (state.temp <= 0){
+        tempAdjustElement.className = "freezing";
+        tempAdjustLandscape.className = "landFreezing";
+    };
```

Comment:
When we have large if-statements where each if-block performs the same operations with slightly different values, it can be worth making a helper function that handles the updates to reduce places we're repeating code and could accumulate small typos. With the helper function, each of the ifs here would call that function passing the desired parameters rather than setting attributes themselves.
```javascript
const tempThemeToggle = () => {

    ...

    if (state.temp >= 32){
        updateLandscapeImage('tooHot', 'landHot');
    } else if ...

}
```

```javascript
const updateLandscapeImage = (category, imageName) => {
     tempAdjustElement.className = category;
     tempAdjustLandscape.className = imageName;
}
```